data/geography/canadian_municipalities.json

```diff
@@ -0,0 +1,606 @@
+{
+    "description": "Top 100 Canadian municipalities by 2011 population",
+    "source": "https:\/\/en.wikipedia.org\/wiki\/List_of_the_100_largest_municipalities_in_Canada_by_population",
```

Comment:
Does the URL need to be escaped? There are others which aren't: https://github.com/dariusk/corpora/search?utf8=%E2%9C%93&q=wikipedia&type=Code